packages/auto-label/src/auto-label.ts

```diff
@@ -34,6 +34,7 @@ import {
 } from '@google-automations/bot-config-utils';
 import {syncLabels} from '@google-automations/label-utils';
 
+const API_NA_LABEL = 'api: N/A';
```

Comment:
Can you add this label automatically to all the repos? You can use `syncLabels`.

The existing usage of syncLabels:
https://github.com/googleapis/repo-automation-bots/blob/0f9806afbc2a9bfd7260df817057d676bd0f2912/packages/auto-label/src/auto-label.ts#L451-L460

Reply:
@tmatsuo, don't the changes in `packages/label-sync/src/labels.json` automatically add it to all repos?

Reply:
That works too, I think. The benefit of syncing the labels is that we can selectively add the label only to relevant repos (repos with the bot config, etc).

Sometimes, repo owners complain when we add labels that they don't want.

Reply:
Oh I see. The only problem I see is that this problem really only arises when auto-label tries to auto-detect a product, which I think happens when a repo does not have a config. So I think in this case its helpful for all repos to have it, but I might be wrong about the logic I describe above ^

Reply:
Got it, maybe we can add the label to `packages/label-sync/src/labels.json` then.